server/bundler/loader.js

```diff
@@ -84,7 +83,7 @@ function splitTemplate( path, section ) {
 		'page( ' + pathRegex + ', function( context, next ) {',
 		'	if ( _loadedSections[ ' + JSON.stringify( section.module ) + ' ] ) {',
 		'		controller.setSection( ' + JSON.stringify( section ) + ' )( context );',
-		'		layoutFocus.next();',
+		'		context.store.dispatch( { type: "LAYOUT_NEXT_FOCUS_ACTIVATE" } );',
```

Comment:
oohhhhhh 😓 I was led astray because the other action creators in this block used raw constants and I thought maybe it was required. Red herring!